homeassistant/components/efergy/sensor.py

```diff
@@ -146,7 +153,7 @@ def __init__(
         if sid:
             self._attr_name = f"efergy_{sid}"
         if description.key == CONF_COST:
-            self._attr_native_unit_of_measurement = f"{currency}/{period}"
+            self._attr_native_unit_of_measurement = currency
```

Comment:
This will be a breaking change. Is just the currency accurate as unit of measurement for this measurement?

If we're going to keep the device class as monetary for this sensor we need to validate that the currency config is an ISO-standard currency code.

https://github.com/home-assistant/core/blob/b1f4ccfd6b69dde1a8ca0ec73db34428e7c76c8e/homeassistant/helpers/config_validation.py#L1287

Reply:
The user can specify the period in the yaml and it defaults to yearly.